<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseFieldDialog</class>
 <widget class="QDialog" name="ChooseFieldDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление конфигурациями</string>
  </property>
  <widget class="QLabel" name="configurationLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Конфигурации:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="configurationLineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="renameButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Переименовать</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="okCancelDialogBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="configurationList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>261</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
